.env.sample

```diff
@@ -3,9 +3,15 @@ BMS_RELEASE=16.0
 BMS_DB_HOST=bmsmysql
 BMS_DB_PASS=CHANGE_PASSWORD
 BMS_CROPS=maize
+BMS_JAVA_OPTS=-server -Xms2g -Xmx2g -XX:+CMSClassUnloadingEnabled -Dspring.profiles.active=development
+
+# Run BMS with liquibase DISABLED
+# BMS_JAVA_OPTS=-server -Xms2g -Xmx2g -XX:+CMSClassUnloadingEnabled
 
 # Variables when running BMS with SSL
 #BMS_SSL_HOST=bmsdock.leafnode.io
 #BMS_SSL_EMAIL=leafnodedev@leafnode.io
 #BMS_SCHEME=https
-#BMS_PORT=443
\ No newline at end of file
+#BMS_PORT=443
+
+BMS_CATALINA_OPTS=-javaagent:/opt/spring-instrument-4.1.6.RELEASE.jar
```

Comment:
No need to specify `BMS_CATALINA_OPTS` in env file as it should not be something configurable. 
We can just hardcode in docker configuration files